MSTuringEmbeddingOnlyScenario.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.Configuration;
+namespace VectorIndexScenarioSuite
+{ 
+    internal class MSTuringEmbeddingOnlyScenario : EmbeddingOnlyScearioBase

```

Comment:
If this class works for both MSTuring and MSMarco can we have the name reflect that?